### PR TITLE
[182689] [FIX] ProFIT returns press sheet's preview PDF link for mailing-mock type press sheets in REST API

### PR DESCRIPTION
**Pull request by andrewreshetnikov**
- [x] [183521](https://farheapsolutions.visualstudio.com/ProFIT-PHP/_workitems/edit/183521): Fix

### CODE REVIEW

**Review by Valeriy-F — APPROVED** [review]

**Comment by Valeriy-F** [comment]
:+1:

File: jquery.littleGallery.js [hunk 1]
```diff
@@ -71,30 +74,20 @@
...
-            if (data.variable_uv_front) {
-                headHtml += '<span class="littleGallery-img" data="' + data.variable_uv_front.url + '">' + data.variable_uv_front.name + '</span>';
-            }
-
-            if (data.variable_uv_back) {
-                headHtml += '<span class="littleGallery-img" data="' + data.variable_uv_back.url + '">' + data.variable_uv_back.name + '</span>';
+            if (!$.isEmptyObject(data)) {
+                var headHtml = '';
+                for (var key in data) {
+                    if (data.hasOwnProperty(key)) {
```
Comment (Valeriy-F) [inline jquery.littleGallery.js]: may be use jQuery.each() (http://api.jquery.com/jquery.each/) and check url and name property existance?

**Review by Valeriy-F — APPROVED** [review]

**Comment by Valeriy-F** [comment]
:+1:

File: jquery.littleGalleryMultipage.js [hunk 2]
```diff
@@ -74,15 +76,21 @@
...
+                if (firstObject.front) {
+                    headHtml += '<span class="littleGallery-img littleGallery-img-front" data="' + firstObject.front + '">Front</span>';
+                }
+                if (firstObject.back) {
+                    headHtml += '<span class="littleGallery-img littleGallery-img-back" data="' + firstObject.back + '">Back</span>';
+                }
+                gallery.find('.littleGallery-head').html(headHtml);
+            } else {
+                gallery.find('.littleGallery-body .littleGallery-container').html(
+                    '<i class="littleGallery-notFound">No preview for PDF VT building</i>'
```
Comment (ngorborukov) [inline jquery.littleGalleryMultipage.js]: It shouldn't be here

**Review by ngorborukov — APPROVED** [review]

**Pull request merged**